### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by aleksandrDrozdovZennolab**

### CODE REVIEW

File: CapMonsterCloud.Client.IntegrationTests/ObjectGen.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    WebsiteKey = Gen.RandomGuid(),
+                    Invisible = Gen.RandomBool(),
+                    Data = Gen.RandomString(),
+                    UserAgent = Gen.UserAgent(),
+                    Cookies = Gen.ListOfValues(Gen.RandomString).ToDictionary(_ => Gen.RandomString(), value => value),
+                    NoCache = Gen.RandomBool()
+                };
+            }
+
+            public static HCaptchaResponse CreateResponse()
```
Comment (azeriker) [inline CapMonsterCloud.Client.IntegrationTests/ObjectGen.cs:25]: CreateSolution should be

File: CapMonsterCloud.Client.IntegrationTests/HCaptchaProxylessTests.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                    }),
+                    ActualResponse: new
+                    {
+                        status = "ready",
+                        solution = new
+                        {
+                            gRecaptchaResponse = captchaSolution.Value,
+                            respKey = captchaSolution.RespKey,
+                            userAgent = captchaSolution.UserAgent,
+                            coockies = captchaRequest.Cookies,
```
Comment (azeriker) [inline CapMonsterCloud.Client.IntegrationTests/HCaptchaProxylessTests.cs:54]: typo

File: CapMonsterCloud.Client.IntegrationTests/Sut.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+namespace CapMonsterCloud.Client.IntegrationTests
+{
+    public class Sut
+    {
+        private Mock<HttpMessageHandler> httpMessageHandler;
+        private ICapMonsterCloudClient cloudClient;
+
+        public List<(RequestType, string)> actualRequests = new();
+
+        public Sut CreatedSut(string clientKey)
```
Comment (azeriker) [inline CapMonsterCloud.Client.IntegrationTests/Sut.cs:24]: CreateSut should be

File: CapMonsterCloud.Client.IntegrationTests/CapMonsterCloud.Client.IntegrationTests.csproj [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
```
Comment (azeriker) [inline CapMonsterCloud.Client.IntegrationTests/CapMonsterCloud.Client.IntegrationTests.csproj:4]: net6

File: CapMonsterCloud.Client/CapMonsterCloudClientFactory.cs [hunk 5]
```diff
@@ -47,7 +47,7 @@ public ICapMonsterCloudClient Create()
         public static ICapMonsterCloudClient Create(ClientOptions options)
             => Create(options, httpMessageHandlerFactory: null, configureClient: null);
 
-        private static ICapMonsterCloudClient Create(
+        internal static ICapMonsterCloudClient Create(
```
Comment (azeriker) [inline CapMonsterCloud.Client/CapMonsterCloudClientFactory.cs:50]: can be leave private. Use CapMonsterCloudClientFactory ctor

File: CapMonsterCloud.Client.IntegrationTests/ObjectGen.cs (hunk 1 above)
Comment (aleksandrDrozdovZennolab) [reply CapMonsterCloud.Client.IntegrationTests/ObjectGen.cs]: done

File: CapMonsterCloud.Client.IntegrationTests/HCaptchaProxylessTests.cs (hunk 2 above)
Comment (aleksandrDrozdovZennolab) [reply CapMonsterCloud.Client.IntegrationTests/HCaptchaProxylessTests.cs]: done

File: CapMonsterCloud.Client/CapMonsterCloudClientFactory.cs (hunk 5 above)
Comment (aleksandrDrozdovZennolab) [reply CapMonsterCloud.Client/CapMonsterCloudClientFactory.cs]: done

**Pull request merged**